config/google_merchant_id_to_gateway_account_id.js

```diff
@@ -0,0 +1,6 @@
+// This config lists the gateway account ids that should use the GOOGLE_PAY_MERCHANT_ID_2
+// environment variable as the merchant id when making a payment request to google.
+
+module.exports = {
+  gateway_account_ids_for_google_pay_merchant_id_2: []
```

Comment:
is the plan to hardcode this list? how would we deal with accounts in different environments?

Reply:
I will use another environment variable for this as per your suggestion.